### PR TITLE
WMCO-3.0.0: Temporarily removed WMCO information from the OCP 4.8 release

### PR DESCRIPTION
**Pull request by EricPonvelle**
Due to WMCO 3.0.0 releasing a few days after OCP 4.8, the WMCO documentation needs to be temporarily hidden until the latest version that is compatible with OCP 4.8 is released.

### CODE REVIEW

File: windows_containers/enabling-windows-container-workloads.adoc [hunk 1]
```diff
@@ -37,5 +34,5 @@ include::modules/configuring-secret-for-wmco.adoc[leveloffset=+1]
 
 == Additional resources
 
-* xref:../installing/installing_azure/installing-azure-default.adoc#ssh-agent-using_installing-azure-default[Generating a key pair for cluster node SSH access]
+ * xref:../installing/installing_azure/installing-azure-default.adoc#ssh-agent-using_installing-azure-default[Generating a key pair for cluster node SSH access]
```
Comment (mburke5678) [inline windows_containers/enabling-windows-container-workloads.adoc:37]: Extra space here??

File: installing/installing_aws/installing-aws-network-customizations.adoc [hunk 2]
```diff
@@ -68,7 +68,7 @@ include::modules/configuring-hybrid-ovnkubernetes.adoc[leveloffset=+1]
 
 [NOTE]
 ====
-For more information on using Linux and Windows nodes in the same cluster, see xref:../../windows_containers/understanding-windows-container-workloads.adoc#understanding-windows-container-workloads[Understanding Windows container workloads].
+For more information on using Linux and Windows nodes in the same cluster, see Understanding Windows container workloads.
```
Comment (mburke5678) [inline installing/installing_aws/installing-aws-network-customizations.adoc:71]: Should you remove this note as the referenced section won't be available for searching? (I assume)

File: installing/installing_azure/installing-azure-network-customizations.adoc [hunk 3]
```diff
@@ -46,7 +46,7 @@ include::modules/configuring-hybrid-ovnkubernetes.adoc[leveloffset=+1]
 
 [NOTE]
 ====
-For more information on using Linux and Windows nodes in the same cluster, see xref:../../windows_containers/understanding-windows-container-workloads.adoc#understanding-windows-container-workloads[Understanding Windows container workloads].
+For more information on using Linux and Windows nodes in the same cluster, see Understanding Windows container workloads.
```
Comment (mburke5678) [inline installing/installing_azure/installing-azure-network-customizations.adoc:49]: Should you remove this note as the referenced section won't be available for searching? (I assume)

File: networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.adoc [hunk 4]
```diff
@@ -14,7 +14,7 @@ Complete any further installation configurations, and then create your cluster.
 [id="configuring-hybrid-networking-additional-resources"]
 == Additional resources
 
-* xref:../../windows_containers/understanding-windows-container-workloads.adoc#understanding-windows-container-workloads[Understanding Windows container workloads]
-* xref:../../windows_containers/enabling-windows-container-workloads.adoc#enabling-windows-container-workloads[Enabling Windows container workloads]
+* Understanding Windows container workloads
+* Enabling Windows container workloads
```
Comment (mburke5678) [inline networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.adoc:18]: Should you remove these two bullets as the referenced sections won't be available for searching? (I assume)

File: windows_containers/windows-containers-release-notes-2-x.adoc [hunk 5]
```diff
@@ -62,7 +62,7 @@ This release adds the following new features and improvements.
 [id="wmco-2-0-0-vsphere-support"]
 ==== Support for clusters running on VMware vSphere
 
-You can now run Windows nodes on a cluster installed on VMware vSphere version 6.5, 6.7, or 7.0. You can create a Windows `MachineSet` object on vSphere to host Windows Server compute nodes. For more information, see xref:../windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.adoc#creating-windows-machineset-vsphere[Creating a Windows `MachineSet` object on vSphere].
+You can now run Windows nodes on a cluster installed on VMware vSphere version 6.5, 6.7, or 7.0. You can create a Windows `MachineSet` object on vSphere to host Windows Server compute nodes. For more information, see Creating a Windows `MachineSet` object on vSphere.
```
Comment (mburke5678) [inline windows_containers/windows-containers-release-notes-2-x.adoc:65]: s/`MachineSet`/MachineSet
Should you remove this xref as the referenced section won't be available for searching? (I assume)

**Comment by mburke5678** [comment]
A few nits. Otherwise LGTM. I didn't check the previews on the changed material. Am basing my LGTM on the PR changes.

Assuming this has not gone through change control, does it need change control where you are hiding content that was in 4.6 and 4.7?
@vikram-redhat ??

**Comment by vikram-redhat** [comment]
This shouldn't be hiding content in 4.6/4.7, only 4.8.

**Comment by vikram-redhat** [comment]
And yeah change control ack is given :)

**Comment by mburke5678** [comment]
> This shouldn't be hiding content in 4.6/4.7, only 4.8.

That is my concern. If users in 4.8 search for the referenced heading, they get the 4.6 and 4.7 docs in the search results. I am wondering if that is acceptable. Or, should we hide those references until the 4.8 version of those docs are available.

**Comment by EricPonvelle** [comment]
Closing this PR as we no longer need to remove the WMCO information from the 4.8 release.

File: windows_containers/windows-containers-release-notes-2-x.adoc [hunk 6]
```diff
@@ -62,7 +62,7 @@ This release adds the following new features and improvements.
 [id="wmco-2-0-0-vsphere-support"]
 ==== Support for clusters running on VMware vSphere
 
-You can now run Windows nodes on a cluster installed on VMware vSphere version 6.5, 6.7, or 7.0. You can create a Windows `MachineSet` object on vSphere to host Windows Server compute nodes. For more information, see xref:../windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.adoc#creating-windows-machineset-vsphere[Creating a Windows `MachineSet` object on vSphere].
+You can now run Windows nodes on a cluster installed on VMware vSphere version 6.5, 6.7, or 7.0. You can create a Windows MachineSet object on vSphere to host Windows Server compute nodes.
```
Comment (skrthomas) [inline windows_containers/windows-containers-release-notes-2-x.adoc:65]: on a cluster that is installed on VMware...

Suggestion to add that. Idk if its grammatically necessary 🤷‍♀️

File: windows_containers/windows-containers-release-notes-2-x.adoc [hunk 7]
```diff
@@ -76,4 +76,4 @@ Windows nodes are now fully integrated with most of the monitoring capabilities
 
 * The Prometheus windows_exporter used by the WMCO currently collects metrics through HTTP, so it is considered unsafe. You must ensure that only trusted users can retrieve metrics from the endpoint. The windows_exporter feature recently added support for HTTPS configuration, but this configuration has not been implemented for WMCO. Support for HTTPS configuration in the WMCO will be added in a future release.
 
-* The kube-proxy service terminates unexpectedly after the load balancer is created if you create the load balancer after the Windows pods begin running. To avoid this issue, you must create the load balancer before the Windows pods status transitions to `Running`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939968[*BZ#1939968*])
\ No newline at end of file
+* The kube-proxy service terminates unexpectedly after the load balancer is created if you create the load balancer after the Windows pods begin running. To avoid this issue, you must create the load balancer before the Windows pods status transitions to `Running`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939968[*BZ#1939968*])
```
Comment (skrthomas) [inline windows_containers/windows-containers-release-notes-2-x.adoc:79]: is windows pods plural, thinking "windows pods statuses" v "status"

**Review by skrthomas — COMMENTED** [review]
Couple of small questions, otherwise LGTM 👍

**Pull request merged**